lib/bosh/BoshDirectorClient.js

```diff
@@ -31,15 +31,16 @@ class BoshDirectorClient extends HttpClient {
     this.oobDirectorConfigs = BoshDirectorClient.getOobDirectorConfigs();
     this.cache = {};
     this.cacheLoadInProgress = false;
-    this.populateCache();
+    this.isReady = this.populateCache();
```

Comment:
`isReady` is a boolean here ?

populateCache() does not return boolean , correct ?

Reply:
its a promise.

Reply:
Ok , should we rename the variable then ? 

As I understand now , it contains the reference to cache wrapped in a promise , correct ?